src/main/java/net/mcreator/ui/minecraft/villagers/JVillagerTradeProfession.java

```diff
@@ -39,7 +41,8 @@
 
 public class JVillagerTradeProfession extends JEntriesList {
 
-	private final JComboBox<String> villager = new JComboBox<>();
+	private final DataListComboBox villagerProffesion = new DataListComboBox(mcreator,
```

Comment:
@KlemenDEV is it a typo? It's `villagerProfession` with only one `f` and two `s`

Reply:
Good catch

Reply:
Should I rename it myself or will you do it?

Reply:
Fixed

Reply:
You marked this as resolved but you hate when people mark yours as resolved lol

Reply:
I don't think you understand of the purpose of this resolved thing. It is for maintainers to make sure they checked and resolved all issues before merging to master.